AllReadyApp/Web-App/AllReady/Areas/AreaNameConstants.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AllReady.Areas
+{
+    public static class AreaNameConstants
+    {
+        public const string ADMIN_AREA_NAME = "Admin";
```

Comment:
Thanks, I also wasn't sure about the location of this class. I looked for similar examples but couldn't find any, feel free to suggest a better location.